spec/views/jobseekers/job_applications/_messages_spec.rb

```diff
@@ -0,0 +1,88 @@
+require "rails_helper"
+
+RSpec.describe "jobseekers/job_applications/_messages.html.slim" do
+  let(:jobseeker) { create(:jobseeker) }
+  let(:job_application) { create(:job_application, :submitted, jobseeker: jobseeker) }
+  let(:messages) { [] }
+
+  before do
+    assign(:job_application, job_application)
+    assign(:show_form, "false")
+    # rubocop:disable RSpec/VerifiedDoubles
+    model_name = double("ModelName", param_key: "publishers_job_application_messages_form")
+    errors = double("Errors", empty?: true, any?: false)
+    message_form = double("MessageForm", model_name: model_name, to_key: nil, persisted?: false, errors: errors, content: "")
+    # rubocop:enable RSpec/VerifiedDoubles
```

Comment:
🙋‍♂️ Since we're disabling a cop here, do we really need to do this? Couldn't we just use `instance_double` with a `Publishers::JobApplication::MessagesForm` instance?

Even if we need to stub its internal methods, would be nice if we could benefit of the `instance_double` check that ensures that the stubbed method exist on the real model.

But maybe there is a need?